Description: Manpage fixes
 Fix first line and use proper escape sequence for (c) character.
Author: Sandro Tosi <matrixhasu@gmail.com>

Index: wmclock-1.0.13/wmclock.man.in
===================================================================
--- wmclock-1.0.13.orig/wmclock.man.in	2010-12-03 21:47:21.000000000 +0100
+++ wmclock-1.0.13/wmclock.man.in	2010-12-04 17:39:17.764387469 +0100
@@ -25,9 +25,7 @@
 .TH wmclock 1x "@DATE@" "Version @VERSION@"
 .\" ----------------------------------------------------------------
 .SH NAME
-.PP
-.BR wmclock :
-a dockable clock for the Window Maker window manager
+wmclock \- A dockable clock for the Window Maker window manager
 .\" ----------------------------------------------------------------
 .SH SYNOPSIS
 .HP
@@ -199,7 +197,7 @@
 .\" ----------------------------------------------------------------
 .SH COPYRIGHT AND LICENSE
 .PP
-Copyright © 1999 by Jim Knoble <jmknoble@pobox.com>.  Significant
+Copyright \(co 1999 by Jim Knoble <jmknoble@pobox.com>.  Significant
 portions of
 .B wmclock
 are directly derived from
